lib/platform/gitlab/index.ts

```diff
@@ -876,10 +862,51 @@ export async function addAssignees(
   }
 }
 
-export function addReviewers(iid: number, reviewers: string[]): Promise<void> {
-  logger.debug(`addReviewers('${iid}, [${reviewers.join(', ')}])`);
-  logger.warn('Unimplemented in GitLab: approvals');
-  return Promise.resolve();
+export async function addReviewers(
+  iid: number,
+  reviewers: string[]
+): Promise<void> {
+  logger.debug(`Adding reviewers '${reviewers.join(', ')}' to #${iid}`);
+
+  if (lt(defaults.version, '13.9.0')) {
+    logger.warn(
```

Comment:
```suggestion
    logger.debug(
```

No warn, like gitea

Reply:
I know gitea does it that way, but doesn't that mean that if someone doesn't have debug logging on, they won't see that reviewers got skipped? And might wonder what went wrong? E.g. if I specify reviewers in my config, I would expect to get at least a warning if it wasn't possible to add them. Or is there something special about the warn handling in renovate? Would info be better?

Reply:
ok, leave it at warn, we will track it in #9420